index.js

```diff
@@ -9,17 +9,34 @@ const tmpDir = require('./tmp-dir');
 
 const inputFileName = core.getInput('file-name');
 const awsOutputDir = core.getInput('aws-output-directory');
-const outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
+const conversionType = core.getInput('conversion-type');
+let outputFileName = inputFileName.replace(/\.xd$/, `.${awsOutputDir}`);
```

Comment:
So you replace `.xd` with `.sketch` here and then in line 16 you add an other `.sketch` to it? I dont think thats correct, can you double check?

Reply:
Done !